test/CMakeLists.txt

```diff
@@ -829,6 +831,22 @@ if(iodaconv_bufr_ENABLED)
                     DEPENDS bufr2ioda.x )
 endif()
 
+
+#==============================================================================
+# Converters for NRT ingest
+#==============================================================================
+
+# do not know how to run with my own python environment (which has h5py)
+ecbuild_add_test( TARGET  test_iodaconv_atms_nc4
+                  TYPE    SCRIPT
+                  ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
+                  COMMAND ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
+                  ARGS    netcdf 
+                          "/home/bruston/.local/external/miniconda3/bin/miniconda3/envs/py3/bin/python
```

Comment:
I think the reason it failed the CI test was because I don't have access to your python directory:
"/home/bruston/.local/external/miniconda3/bin/miniconda3/envs/py3/bin/python
can you please try replace that with ${Python3_EXECUTABLE}, perhaps that 'll fix the CI test issue.